src/Sylius/Component/Core/Model/Order.php

```diff
@@ -318,6 +318,20 @@ public function getLastPayment($state = null)
     }
 
     /**
+     * @return bool
+     */
+    public function requiresShipping()
```

Comment:
@Persata yeah, I've suggested the first name that came to my mind, that usually isn't the best one :D